config/locales/models.fr.yml

```
@@ -928,6 +930,7 @@ fr:
     solicitation_full_name: Contact en entreprise
     solicitation_gclid: Id GA
     solicitation_id: Id de la sollicitation
+    solicitation_origin_id: Id de la page d'origine
```

Le petit apostrophe courbe, bon ça c'est histoire de faire un commentaire

```suggestion
    solicitation_origin_id: Id de la page d’origine
```